src/content/docs/en/guides/media.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Use a DAM with Astro
+description: How to use a Digital Asset Manager (DAM) to add images and videos to Astro
+i18nReady: true
+---
+import MediaGuidesNav from '~/components/MediaGuidesNav.astro';
+import ReadMore from '~/components/ReadMore.astro';
+import Badge from "~/components/Badge.astro"
+
+**Ready to connect a headless Digital Asset Manager (DAM) to your Astro project?** Follow one of our guides to integrate a hosted media system.
+
+:::tip
+Find [community-maintained integrations](https://astro.build/integrations/?search=image) for connecting a DAM or hosted media system to your project in our integrations directory. 
```

Comment:
```suggestion
Find [community-maintained integrations](https://astro.build/integrations/) for connecting a DAM or hosted media system to your project in our integrations directory. 
```
Going to take out the search for now because there isn't a single good word that seems to pull up the most relevant results!